### DISCUSSION TITLE
[SOLVED] Using two COUNT inside SELECT returns the same value (SQL)

### DISCUSSION THREAD

**Discussion by Delthoid** [supabase/supabase#7165] [Questions]
Okay so here in my result as you can see the c_count has all the same value as r_count, but r_count is coming from different table.
What went wrong?

Sorry I'm still learning SQL things.
Thank you

![image](https://user-images.githubusercontent.com/53704269/171998701-ef8f89e8-54fa-49ba-bdbd-83456c9aaffa.png)

**Comment by jvbianchi** [comment]
Hi, instead of using COUNT, have you tried using SUM?

**Reply by Delthoid** [reply]
Like this? 
It shows all zeros

![image](https://user-images.githubusercontent.com/53704269/172013659-b87eff1d-df73-4042-96f3-a9653a577601.png)
